kolibri-electron/src/index.js

```diff
@@ -27,7 +27,7 @@ let django = null;
 
 let KOLIBRI_HOME_TEMPLATE = '';
 let KOLIBRI_EXTENSIONS = '';
-let KOLIBRI_HOME = path.join(os.homedir(), '.endless-key');
+let KOLIBRI_HOME = path.join(os.homedir(), '.endless-key-iguana');
```

Comment:
I see 2 issues with this solution:
- We will have to remember to update this string on each release. It has to at least be a constant and the process documented somewhere, but as it is now, is not future-proof.
- Is not good for our users to leave hidden folders in each release. But I guess the chances for this to happen are few (as they are few the chances of someone using the previous release after this one).